packages/app/src/runner/screenshot/screenshot.cy.tsx

```diff
@@ -37,6 +37,16 @@ const Layout: FunctionalComponent = () => {
 
 const captureTypes = ['fullPage', 'viewport', 'runner'] as const
 
+function removeGlobalStyles () {
+  cy.get('style').each((item) => {
+    if (item[0].dataset.cy !== 'injected-style-tag') {
+      item.remove()
+    }
+  })
+
+  return cy.get('style').should('have.length', 1)
```

Comment:
This now means that only the existing CSS injected by the component will be applied. Other CSS targeting body, HTML, global reset CSS etc will not apply here. 

We may want to change this later, but for now this gives us the clearest diff with 10.0-release.

Reply:
>This now means that only the existing CSS injected by the component will be applied. 
This comment has great context!! What do you think about including it in the code?